test/utils/index.js

```diff
@@ -2,7 +2,10 @@
 
 const PeerId = require('peer-id')
 const PeerInfo = require('peer-info')
-const Node = require('./nodejs-bundle')
+const isNode = require('detect-node')
+
+const Node = isNode ? require('./nodejs-bundle') : require('./browser-bundle')
```

Comment:
You could just use the browser field in package.json to do the check, similar to https://github.com/libp2p/js-libp2p/blob/master/package.json#L38. It will be auto swapped on the browser build step.

Reply:
yes, with that I can get rid of the `detect-node`!